client/client_test.go

```diff
@@ -826,6 +845,20 @@ func fakeServerData(t *testing.T, repo *NotaryRepository, mux *http.ServeMux,
 			targetsJSON, _ := json.Marshal(signedTargets)
 			fmt.Fprint(w, string(targetsJSON))
 		})
+
+	mux.HandleFunc("/v2/docker.com/notary/_trust/tuf/targets/level1.json",
+		func(w http.ResponseWriter, r *http.Request) {
+			level1JSON, err := json.Marshal(signedLevel1)
+			assert.NoError(t, err)
+			fmt.Fprint(w, string(level1JSON))
+		})
+
+	mux.HandleFunc("/v2/docker.com/notary/_trust/tuf/targets/level2.json",
+		func(w http.ResponseWriter, r *http.Request) {
+			level1JSON, err := json.Marshal(signedLevel2)
```

Comment:
nit: `level2JSON`?  This is non-blocking since it shouldn't matter with the function scope though, but it might be nice for clarity.